src/figuresClasses.ts

```diff
@@ -1,11 +1,95 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
```

Comment:
The `shape` property should be defined as a constant string 'triangle' instead of using a type annotation. It should be `shape = 'triangle';` to match the `Figure` interface.